### ISSUE TITLE
Breakpoints are not hit on Windows

### ISSUE THREAD

**Issue by DominicKramer** [googleapis/cloud-debug-nodejs#795]
Breakpoints don't appear to be hit on Windows (Tested on Windows Server 2019 with Node 12.13.1).

The following code isolates how the debug agent uses V8's inspector API.  Either we are using the API incorrectly or there is a bug in the API.

index.js
```js

const inspector = require('inspector');

const session = new inspector.Session();
session.connect();
session.post('Debugger.enable');
session.post('Debugger.setBreakpointsActive', {active: true});
session.on('Debugger.paused', message => {
	console.log('Paused with message', message);
});

const url = `file://${__filename}`
const line = 26;
const column = 1;

const options = {
  lineNumber: line - 1,
  url,
  columnNumber: column - 1,
  condition: undefined,
};

console.log('Setting breakpoing with options=', options);

session.post('Debugger.setBreakpointByUrl', options, (error, response) => {
  console.log('Setting breakpoint, error=', error, ', response=', response);
});

function f() {
	console.log('f() called');
	const a = 42;
}

setInterval(f, 1000);

```

Running `node index.js` on Linux causes the `Paused with message ...` callback to correctly get called.  When running the code on Windows, the callback is never invoked.  I think this is why breakpoints are not being hit by the debug agent on Windows.

The following is the output on Linux
```
$ node index.js 
Setting breakpoing with options= {
  lineNumber: 25,
  url: 'file:///xxxxx/index.js',
  columnNumber: 0,
  condition: undefined
}
Paused with message {
  method: 'Debugger.paused',
  params: {
    callFrames: [
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object]
    ],
    reason: 'other',
    hitBreakpoints: [
      '1:25:0:file:///xxxxx/index.js'
    ]
  }
}
Setting breakpoint, error= null , response= {
  breakpointId: '1:25:0:file:///xxxxx/index.js',
  locations: [ { scriptId: '51', lineNumber: 25, columnNumber: 2 } ]
}
f() called
f() called
f() called
f() called
```
while on Windows the output is
```
> node index.js
Setting breakpoing with options= {
  lineNumber: 24,
  url: 'file://C:\\xxxxx\\index.js',
  columnNumber: 0,
  condition: undefined
}
Setting breakpoint, error= null , response= {
  breakpointId: '1:24:0:file://C:\\xxxxx\\index.js',
  locations: []
}
f() called
f() called
f() called
f() called
f() called
```
Notice that the `Setting breakpoint` response returns a list of locations with zero elements.

**Comment by DominicKramer** [comment]
@hashseed I was wondering if you could help me out.  Based on the repro above, is there a problem with how we are using the inspector API on Windows.  Thanks.

**Comment by DominicKramer** [comment]
I have a potential fix in #815.  The PR still has unit test failures, but I have verified that the modification I made works to debug on Windows 10.

It seems the `inspector.setBreakpointByUrl` method requires the url to start with `file:///` (not `file://`, note the difference in the number of slashes).  In addition, the url has to have all of its backslashes converted to forward slashes.
